pkg/api/snapshot_resp.go

```diff
@@ -1,14 +1,18 @@
 package api
 
-import "github.com/pborman/uuid"
+import (
+	"time"
+
+	"github.com/pborman/uuid"
+)
 
 // SnapInfo contains static information about the snapshot.
 // Clients should NOT use this struct directly.
 type SnapInfo struct {
 	VolInfo       VolumeInfo `json:"snapinfo"`
 	ParentVolName string     `json:"parentname"`
 	Description   string     `json:"description"`
-	SnapTime      string     `json:"snaptime"`
+	CreatedAt     time.Time  `json:"created-at"`
```

Comment:
This will be serialised to a particular time format chosen by json package. For now, this is okay. In the long term, for other language consumers, it's good to switch to absolute/epoch time.